UnityProject/Assets/Scripts/Items/Weapons/Gun.cs

```diff
@@ -172,8 +172,9 @@ public class Gun : NetworkBehaviour, IPredictedCheckedInteractable<AimApply>, IC
 		/// <summary>
 		/// If true, displays a message whenever a gun is shot
 		/// </summary>
-		[SyncVar(hook = nameof(SyncIsSuppressed)), Tooltip("If the gun displays a shooter message")]
-		public bool isSuppressed;
+		[SerializeField, SyncVar(hook = nameof(SyncIsSuppressed)), Tooltip("If the gun displays a shooter message")]
+		private bool isSuppressed;
+		public bool IsSuppressed => isSuppressed;
```

Comment:
this but with ``isSuppressible`` which I don't expect to change after being set, ever.